InfiniteLoopLMSSQL/tests/daos/AuthorDaoTest.java

```diff
@@ -0,0 +1,105 @@
+package daos;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Test;
+
+import com.st.lmssql.dao.GenericDao;
+import com.st.lmssql.daoImp.AuthorDaoImp;
+import com.st.lmssql.models.Author;
+import com.st.lmssql.utils.ConnectionFactory;
+
+public class AuthorDaoTest {
+
+	private static Connection con = ConnectionFactory.getMyConnection();
+	private static GenericDao<Author> genDaoAuthor = new AuthorDaoImp(con);
+	
+	private int authTestId = 0;
+	
+	
+	//do this after all test cases
+	@After
+	public void cleanUp() throws IOException, SQLException {
+		Author clean = new Author();
+		clean.setAuthorId(authTestId);
+		genDaoAuthor.delete(clean);
+		con.commit();
+	}
+	
+	@Test
+	public void addTest() throws SQLException {

```

Comment:
Name test methods such that "test" is the verb, e.g. this method should be `testAdd`.